assets/airports/kcvg.json

```diff
@@ -0,0 +1,6771 @@
+                ["CAMUM", "ZEGUL", "SILKS"],
+                ["FANSA", "SILKS"],
+                ["BONLE", "CULED"],
+                ["HOBAL", "BOUMR", "CULED"],
+                ["HAGSS", "EVAME", "CEMLA", "FRTZY", "CULED", "SILKS", "TRFWA*"]
+            ]
+        }
+    },
+    "stars": {
+        "CEGRM4": {
+            "icao": "CEGRM4",
+            "name": "Cegram Four",
+            "entryPoints": {
+                "JOT": ["JOT", ["CADIZ", "A290"], ["FATOX", "A240"]]
+            },
+            "body": [["BVT", "A240"], ["TORIO", "A240"], ["MACES", "A190"], ["VHP", "A140"], ["SHB", "A140"], ["CEGRM", "A110"], "BCATT"],
+            "rwy": {
+                "09": ["BAMGE"],
+                "18C": ["BAMGE"],
+                "18L": ["BAMGE"],
+                "18R": ["BAMGE"],
+                "27": ["BAMGE"],
+                "36C": ["BAMGE", "FIDEK", "ZETIP"],
+                "36L": ["BAMGE", "FIDEK", "ZETIP"],
+                "36R": ["BAMGE", "FIDEK", "ZETIP"]
+            },
+            "draw": ["JOT", "CADIZ", "FATOX", "BVT", "TORIO", "MACES", "VHP", "SHB", "CEGRM", "BCATT", "BAMGE", "FIDEK", "ZETIP"]
+        },
+        "GAVNN4": {
+            "icao": "GAVNN4",
+            "name": "Gavin Four",
+            "entryPoints": {
+                "HNN": ["HNN"]
+            },
+            "body": [["JAYKB", "A240"], ["BRUSH", "A100"], ["GAVNN", "A100"], ["KNEDY", "A100"]],
+            "rwy": {
+                "09": ["AAJAY"],
+                "18C": ["AAJAY", "BACAS", "OGIRE"],
+                "18L": ["AAJAY", "BACAS", "OGIRE"],
+                "18R": ["AAJAY", "BACAS", "OGIRE"],
+                "27": ["AAJAY"],
+                "36C": ["AAJAY"],
+                "36L": ["AAJAY"],
+                "36R": ["AAJAY"]
+            },
+            "draw": ["HNN", "JAYKB", "BRUSH", "GAVNN", "KNEDY", "AAJAY", "BACAS", "OGIRE"]
+        },
+        "JAKIE4": {
+            "icao": "JAKIE4",
+            "name": "Jakie Four",
+            "entryPoints": {
+                "VXV": ["VXV"]
+            },
+            "body": [["ARTUR", "A190"], ["CHEDA", "A100"], ["ODABE", "A100"], ["HAYZL", "A100"], ["JAKIE", "A100"]],
+            "rwy": {
+                "09":  ["FLM"],
+                "18C": ["FLM", "FIDEK", "BAMGE", "AGEBE"],
+                "18L": ["FLM", "FIDEK", "BAMGE", "AGEBE"],
+                "18R": ["FLM", "FIDEK", "BAMGE", "AGEBE"],
+                "27":  ["FLM"],
+                "36C": ["FLM"],
+                "36L": ["FLM"],
+                "36R": ["FLM"]
+            },
+            "draw": ["VXV", "ARTUR", "CHEDA", "ODABE", "HAYZL", "JAKIE", "FLM", "FIDEK", "BAMGE", "AGEBE"]
+        },
+        "SARGO3": {
+            "icao": "SARGO3",
+            "name": "Sargo Three",
+            "entryPoints": {
+                "PXV": ["PXV"]
+            },
+            "body": [["JRICO", "A140"], ["APCOT", "A100"], ["JKUKA", "A100"], ["SARGO", "A100"], ["TYOTA", "A100"]],
```

Comment:
APCOT has a FL210 restriction, other restrictions should be much higher except TYOTA. FIDEK could be in body and has a 9,000ft restriction.